package.json

```diff
@@ -25,6 +25,7 @@
     "docker-build": "yarn tsc && yarn workspace example-backend build --build-dependencies && yarn workspace example-backend build-image",
     "backstage-create": "backstage-cli create --scope backstage --no-private",
     "create-plugin": "yarn backstage-create --select plugin",
+    "create-backend-plugin": "yarn backstage-create --select backend-plugin",
```

Comment:
We don't really want to add this tbh, as we're migrating away from `create-plugin`.

The one migration we could get started though is to recommend the creation of a `backstage-create` script in all apps, as well as adding it to the `create-app` template. Then we could update the docs to point to `yarn backstage-cli --select backend-plugin` instead. If that's something you'd like to pick up

Reply:
Ok, I'll take a look at that.

Reply:
Hey, just wanted to check in if this is still on your radar :)